### ISSUE TITLE
the number of epochs

### ISSUE THREAD

**Issue by godhj93** [zouchuhang/LayoutNetv2#10]
Hi, i am training the network as you described in this repository.

I found that the number of epochs set 99999. 

is there any reason to set it 99999?

I am wondering if I can set the number of epochs 100 since it takes too much time to train.

thanks :)

**Comment by zouchuhang** [comment]
The training script will save the model only if the validation loss achieves the smallest. So you can stop the training job at any time when you feel that the validation loss will never go down.
